### PR TITLE
Feature/lab discussion

### PR DESCRIPTION
**Pull request by lewisdaly**
Adds the workbench/lab discussion document. 

This document is not meant to be complete just yet, but I wanted to get it added to the documentation to mark a milestone in the OSS Lab investigation.

### CODE REVIEW

File: discussions/workbench.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+There are others in the community working on some of these needs we outlined above. How can we align ourselves together to: (1) Not duplicate efforts (nor step on each other's toes) and (2) Provide the most impact for end users and the Mojaloop community as a whole
+
+In general, we reached a consensus around the following:
+- any OSS Lab effort should be focused with a specific end user in mind
+- Our focus should be further out on the mojaloop spokes (DFSPs, Fintechs, 3rd Party app providers)
+
+
+### MIFOS
+- Already extensive work done here with Finteract system, which provides out-of-the-box solution for Mojaloop enabled DFSPs
```
Comment (NicoDuvenage) [inline discussions/workbench.md]: very minor - spelling=Fineract

File: discussions/workbench.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+### ModusBox
+- More on the systems integrator perspective. Building a bunch of tools already to ease the development and onboarding process for switches and DFSPs
+- Have open sourced the Mojaloop JS SDK
+- Interested in showing 'how the engine works' to build confidence in business parters/customers
+- Also interested (especially in WOCCU case) as a Mojaloop lab as a place for Fintechs to learn and test concepts on top of Mojaloop Switches
+  - Once this is connected, the interesting use cases will start to develop beyond tranfers from A to B
+  - MFIs (especially small-medium) don't have much capacity for experimentation or developing new business cases, but these cases can be driven from fintechs first
+- How can we assist orgs. who have little-to-no technical capability to become confident with Mojaloop?
+  - A technical lab environment won't do much in this case
+  - Can we demonstrate Mojaloop over a spreadsheet? Everyone can undertand spreadsheets.
```
Comment (NicoDuvenage) [inline discussions/workbench.md]: understand

File: discussions/workbench.md (hunk 1 above)
Comment (lewisdaly) [reply discussions/workbench.md]: Haha that wasn't even a typo - I genuinely thought it was "Finteract"...

File: discussions/workbench.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+
+## 1. Goals for the PI8 Convening:
+
+1. Define terms and outline assumptions
+2. Outline existing efforts and how the OSS Community aligns with them (GSMA, MIFOS, ModusBox)
+3. Define users and usecases, and exclude the users we won't worry about
+4. Recommendations for a few different solutions to the "Lab Problem"
+	- Documentation around business cases and personas Dan developed
+  - Simple Mojaloop-over-spreadsheets demo, to get people using mojaloop without Postman
```
Comment (elnyry) [inline discussions/workbench.md]: Was this intentional or a formatting issue?

File: discussions/workbench.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+- 4.2 Pilots train with simulators _before_ flying a real life, dangerous and expensive plane.
+- 4.3 Within Mojaloop: a simulator can simulate interacting with some component of the system
+  - Replace an entire switch to test a DFSP implementation
+  - Simulate 2 DFSPs to test a switch deployment
+  - A simulator also reduces the need for someone to be with the person testing. So a DFSP can send and receive via the switch, without interaction with the Hub Operator.
+
+
+## 3. Assumptions
+
+>_Some of these may go without saying, but it's noting them here anyway._
```
Comment (elnyry) [inline discussions/workbench.md]: Formatting on the points below

**Comment by lewisdaly** [comment]
@elnyry should be good to go again with those formatting fixes. Unfortunately to get nested bullets working with lines such as 4.1.2, we need to have both bullets and numbers.

**Comment by lewisdaly** [comment]
Thanks all for your helpful feedback and suggestions.

**Pull request merged**